docs/guide/images.md

```diff
@@ -205,12 +205,12 @@ In addition to [filters exposed by LiipImagineBundle](http://symfony.com/doc/mas
 | geometry/scalepercent        | \[widthPercent, heightPercent\]             | Scales width and height with provided percent values. Proportion will not be kept.                 |
 | geometry/crop                | \[width, height, startX, startY\]           | Crops the image. Result will have provided width/height, starting at provided startX/startY        |
 | border                       | \[thickBorderX, thickBorderY, color=\#000\] | Adds a border around the image. Thickness is defined in px. Color is "\#000" by default.           |
-| filter/noise                 | \[radius=0\]                                | Smooths the contours of an image (`imagick`/`gmagick` only). `radius` is in pixel.                 |
-| filter/swirl                 | \[degrees=60\]                              | Swirls the pixels of the center of an image (`imagick`/`gmagick` only). `degrees` defaults to 60°. |
+| filter/noise                 | \[radius=0\]                                | Smooths the contours of an image (`imagick`/`gmagick` only). `radius` is in px.                    |
```

Comment:
Why this change (pixels in px)?

Reply:
To be consistent with "px" that is used in the earlier row in the same table.